src/GuestConfiguration/GuestConfiguration/Cmdlets/GetAzVMGuestPolicyStatusHistory.cs

```diff
@@ -24,18 +24,16 @@ namespace Microsoft.Azure.Commands.GuestConfiguration.Cmdlets
     /// <summary>
     /// Gets Vm Guest Policy reports (GuestConfiguration policy reports)
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "VMGuestPolicyStatusHistory", DefaultParameterSetName = ParameterSetNames.VmNameScope)]
```

Comment:
Set DefaultParameterSetName = InitiativeIdScope

Reply:
hey, we want users to be able to use the below by default. (for ex. if they hit Get-AzVMGuestPolicyStatusHistory without parameters.
Get-AzVMGuestPolicyStatusHistory [-ResourceGroupName] <System.String> [-VMName] <System.String>
Making InitiativeIdScope default will make users to enter the InitiativeId

Reply:
Because the InitiativeId is optional, making the InitiativeIdScope will not require the users to enter the IntitativeID

Reply:
done